open-codegen/opengen/icasadi/extern/icasadi.c

```diff
@@ -1,3 +1,5 @@
+/* NOTE: This file is only used for testing. */
```

Comment:
Do we need these files? (`icasadi.c`, `icasadi.h` and `icasadi_config.h`).

Reply:
I remember you said you used them for testing, else I will remove them

Reply:
I use a particular instance of `icasadi` for testing; this is already generated once and it's used in Rust. I'm referring to the following lines in the main `Cargo.toml`:

```toml
# --------------------------------------------------------------------------
# T.E.S.T.   D.E.P.E.N.D.E.N.C.I.E.S
# --------------------------------------------------------------------------
# These dependencies are only used for testing
[dev-dependencies]
unit_test_utils = "0.1.3"
# Note: path dependencies (even dev-dependencies) cannot be local in order
#       to publish a crate, so the following line can only be used when
#       testing locally
# icasadi_test = { path = "test/icasadi_test/" }
#       instead, use:
icasadi_test = "0.0.2"
```
You can remove these files.